### ISSUE TITLE
MetaInfo interface does not contain titleChunk key

### ISSUE THREAD

**Issue by vintprox** [nuxt/vue-meta#557]
`titleChunk` is not augmented by default in `MetaInfo` interface. That easily confuses TypeScript developer, trying to retrieve original title of current page, be it in `changed` callback or direct `$meta().refresh()` - TypeScript will complain that `titleChunk` doesn't exist, for good.

**Comment by vintprox** [comment]
Manually augmenting interface saves the day:
```
declare module 'vue-meta' {
  export interface MetaInfo {
    titleChunk?: string
  }
}
```
But that would be most convinient to see implemented in `vue-meta`

**Comment by pimlie** [comment]
@vintprox If you already know/have the fix, please feel free to submit a PR for it :)

**Comment by vintprox** [comment]
Should be solved with #559
